tinygrad/nn/__init__.py

```diff
@@ -34,19 +34,20 @@ def __call__(self, x:Tensor):
     return x.batchnorm(self.weight, self.bias, batch_mean, batch_invstd)
 
 # TODO: is this good weight init?
+# kaiming initialization is added as default
 class Conv2d:
-  def __init__(self, in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True):
+  def __init__(self, in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True, initialization: str='kaiming_uniform'):
     self.kernel_size = (kernel_size, kernel_size) if isinstance(kernel_size, int) else (kernel_size[0], kernel_size[1])
     self.stride, self.padding, self.dilation, self.groups = stride, padding, dilation, groups
-    self.weight = Tensor.glorot_uniform(out_channels, in_channels//groups, self.kernel_size[0], self.kernel_size[1])
+    self.weight = Tensor.kaiming_uniform(out_channels, in_channels//groups, self.kernel_size[0], self.kernel_size[1]) if initialization=='kaiming_uniform' else Tensor.glorot_uniform(out_channels, in_channels//groups, self.kernel_size[0], self.kernel_size[1])
```

Comment:
getattr(Tensor, initialization)?

Reply:
Great idea, thank you, changed them to getattr(Tensor, initialization) version

Reply:
and fixed some silly typos (: